### ISSUE TITLE
spawn E2BIG error

### ISSUE THREAD

**Issue by ericandrewlewis** [lydell/source-map-visualize#4]
Hi, thanks for building this! I'm excited to use it.

I get an error when I attempt to run it against a medium large sized project:

```
$ ./source-map-visualize ../v0.js.br
internal/child_process.js:366
    throw errnoException(err, 'spawn');
    ^

Error: spawn E2BIG
    at ChildProcess.spawn (internal/child_process.js:366:11)
...
```

Googling around tells me that this has to do with the size of the arguments passed to a child process, so I should note that my ARG_MAX is set to 2097152.

**Comment by lydell** [comment]
Yeah, I've seen this too. The tool pretty much only works with smaller files :(
